usaspending_api/search/v2/views/spending_by_category_views/spending_by_recipient.py

```diff
@@ -72,7 +74,7 @@ def build_elasticsearch_result(self, response: dict) -> List[dict]:
                     "amount": int(bucket.get("sum_field", {"value": 0})["value"]) / Decimal("100"),
                     "recipient_id": recipient_info["hash_with_level"] or None,
                     "name": recipient_info["name"] or None,
-                    "code": recipient_info["unique_id"] or "DUNS Number not provided",
+                    "code": recipient_info["unique_id"] or "Recipient not provided",
```

Comment:
This also broke some tests in `usaspending_api/search/tests/integration/spending_by_category/test_spending_by_recipient.py` because of the expected result changing from `DUNS Number` to `Recipient`

Reply:
Yep, working this now.